docs/docsite/rst/user_guide/collections_using.rst

```diff
@@ -270,6 +270,20 @@ In addition to the ``namespace.collection_name:version`` format, you can provide
 
 Verifying against ``tar.gz`` files is not supported. If your ``requirements.yml`` contains paths to tar files or URLs for installation, you can use the ``--ignore-errors`` flag to ensure that all collections using the ``namespace.name`` format in the file are processed.
 
+If a collection has been signed by the Galaxy server, the server will provide ASCII armored, detached signatures to verify the authenticity of the MANIFEST.json before using it to verify the collection's contents. You must opt into signature verification by configuring a keyring for ``ansible-galaxy`` to use and providing the path with the ``--keyring`` option.
+
+If a keyring is provided, additional signature sources can be included in the ``requirements.yml`` file for each collection by using "signatures" key or for collection name(s) provided on the CLI by using the ``--signature`` option. Signature sources should be URIs.
```

Comment:
```suggestion
If a keyring is provided, additional signature sources can be included in the ``requirements.yml`` file for each collection by using "signatures" key  in a requirements file or for collection name(s) provided on the CLI by using the ``--signature`` option. Signature sources should be URIs.
```

Reply:
@samccann did you mean to remove the earlier mention of the requirements file? I'm mentioning it needs to be in the requirements file twice now, it seems redundant.

Reply:
I re-worked this section by breaking the option for CLI collection name(s) and the option for the requirements file into separate sentences and added an example for the requirements file.